core/platform/taskqueue/gae_taskqueue_services.py

```diff
@@ -36,6 +39,16 @@ def defer_to_events_queue(fn, *args, **kwargs):
     # details on the _queue kwarg.
     deferred.defer(fn, *args, _queue=QUEUE_NAME_EVENTS, **kwargs)
 
+def add_feedback_message_email_task(user_id):
+    """Adds a new task for sending feedback message email."""
+    # See https://cloud.google.com/appengine/docs/python/taskqueue for
+    # details of various parameters set when adding a new task.
+    taskqueue.add(
```

Comment:
I think the oppia-specific parts of this (that refer to user_id, urls in Oppia, etc.) need to go one level higher into the domain ('logic') layer -- that's why the current file has only very specific functions. This is because the platform layer is meant to just be a seam between the application and GAE; it shouldn't know anything about the application logic.

Perhaps in feedback_services.py?